packages/sanity/src/core/form/inputs/files/FileInput/FileActionsMenu.tsx

```diff
@@ -126,7 +129,7 @@ export function FileActionsMenu(props: Props) {
             constrainSize
           >
             <Button
-              aria-label="Open file options menu"
+              aria-label={t('inputs.files.image.actions-menu.files.aria-label')}
```

Comment:
This key doesn't make sense to me...
Multiple `file` and also an `image`?

Reply:
Yeah, I was going by what was already in the resource file. I think @bjoerge was following the directory path, which makes some sense.

Reply:
yeah, I was mirroring the way the files are organized, but agree it reads weird. `inputs.files.image.actions-menu…` should suffice.

Reply:
I see what I did and what made it extra confusing was this key having 'image' in it at all. 😬 . I fixed this one and the others ones to remove 'files'

Reply:
Ah! It was in the _File_ component, didn't catch that.